rest/route/github.go

```diff
@@ -223,12 +223,12 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 				"owner":   owner,
 				"repo":    repo,
 				"item":    PRNum,
-				"message": "PR contains no base branch label",
+				"message": "PR contains no base branch ref",
 			}))
 			return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "PR contains no base branch label"))
 		}
 
-		baseBranch := *pr.Base.Label
+		baseBranch := *pr.Base.Ref
```

Comment:
Another mismatch with the GitHub API: what we really wanted was ref (e.g. master) not label (e.g. _user_.master)